esp8266-fastled-webserver.ino

```diff
@@ -921,6 +936,14 @@ void loadSettings()
 
   showClock = EEPROM.read(9);
   clockBackgroundFade = EEPROM.read(10);
+
+  currentTimeZoneIndex = EEPROM.read(11);
```

Comment:
This would need updates if accepting the above recommendations.

In short:
Store UTC offset directly.
Store DST as an enumeration corresponding to a fixed list of algorithms.